### PR TITLE
#15: RIPE Blender API setup and useful methods

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-orchestra/issues/15 |
| Dependencies | -- |
| Decisions | - Project setup<br>- Methods to endpoints /load and /render |
| Animated GIF | -- |

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.DS_Store
+
+yarn.lock
+example.env
+package-lock.json
```
Comment (joao-conde) [inline .gitignore:5]: Maybe swap

File: js/load.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const LoadAPI = superclass =>
+    class extends superclass {
+        async loadFile(file, options = {}) {
+            const url = this.invoicingBaseUrl + "load";
```
Comment (joao-conde) [inline js/load.js:4]: this.base_url

File: js/render.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+export const RenderAPI = superclass =>
+    class extends superclass {
+        async render(options = {}) {
+            const url = this.invoicingBaseUrl + "render";
```
Comment (joao-conde) [inline js/render.js:4]: this.base_url

File: package.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
+{
+    "name": "ripe-blender-api",
+    "version": "0.1.0",
+    "description": "The RIPE Invoice Express API client",
```
Comment (joao-conde) [inline package.json:4]: The RIPE Blender API client

File: rollup.config.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    " *\n" +
+    " * This source code is licensed under the Apache 2.0 license found in the\n" +
+    " * LICENSE file in the root directory of this source tree.\n" +
+    " */";
+
+export default [
+    {
+        input: "js/index.js",
+        output: {
+            name: "ripeInvoiceExpress",
```
Comment (joao-conde) [inline rollup.config.js:28]: ripeBlender

**Review by joao-conde — CHANGES_REQUESTED** [review]
Reviewing, but I will take care of the fixes since @BeeMargarida is on a well-deserved vacation

**Review by joao-conde — APPROVED** [review]

**Review by 3rdvision — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**